File: h2.py
n=int(raw_input());
for i in range(n):
    arr = [int(x) for x in raw_input().split()]
for i in range(n):
    arr1 = [int(x) for x in raw_input().split()]
arr.sort()
arr1.sort()
for i in range(n):
    result+=abs(arr[i]-arr1[i]);
print result;

File: h2.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int n;
	scanf("%d",&n);
	long long int s[n];
	long long int d[n];
	long long int result=0;
	for(int i=0;i<n;i++)
	{
		scanf("%lld",&s[i]);
	}
	for(int i=0;i<n;i++)
	{
		scanf("%lld",&d[i]);
	}
	sort(s,s+n);
	sort(d,d+n);
	for(int i=0;i<n;i++)
	{
		result+=abs(s[i]-d[i]);
	}
	printf("%lld\n",result);
}